client/src/containers/overview/table/view/key-costs/index.tsx

```diff
@@ -183,6 +179,14 @@ export function KeyCostsTable() {
                         ? (cell.getValue() as string)
                         : undefined
                     }
+                    style={
+                      cell.column.id === "projectName"
+                        ? {
+                            minWidth: "fit-content",
+                            maxWidth: "100%",
```

Comment:
Any reason to not use tailwind classes here?

Reply:
Applying tailwind classes directly didn't work here